serialization/compact_config.go

```diff
@@ -25,10 +25,13 @@ import (
 	"github.com/hazelcast/hazelcast-go-client/types"
 )
 
+// CompactConfig contains compact serializers.
 type CompactConfig struct {
 	serializers map[string]CompactSerializer
 }
 
+// Clone creates a copy of the CompactConfig value.
+// This method is intended for internal use.
 func (cc *CompactConfig) Clone() CompactConfig {
```

Comment:
If the intention is to use internally, is there anything disallow you to make it internal?

Reply:
No, there's not. It's not dangerous to call it by the user though.

Reply:
Np for me, I raised this since other clients doesn't have that public API.

Reply:
We have this method for all types that represent configuration sections.